src/chat/ContactSheet.tsx

```diff
@@ -44,65 +45,104 @@ const ContactSheet = forwardRef<View, Props>(({close}, focusRef) => {
       </View>
 
       <FullScreenFooter>
-        <View
-          accessible={true}
-          ref={focusRef}
-          style={styles.descriptionSection}
-        >
-          <ThemeText type="body__secondary" color="secondary">
-            {t(ContactSheetTexts.customer_service.title)}
-          </ThemeText>
-          <ThemeText>{t(ContactSheetTexts.customer_service.body)}</ThemeText>
-        </View>
-        <Button
-          color="primary_2"
-          text={t(ContactSheetTexts.customer_service.button)}
+        <ContactItem
+          title={t(ContactSheetTexts.customer_service.title)}
+          body={t(ContactSheetTexts.customer_service.body)}
+          buttonText={t(ContactSheetTexts.customer_service.button)}
+          focusRef={focusRef}
+          icon={() => (
+            <ThemeIcon colorType="primary_2" svg={Support}></ThemeIcon>
+          )}
           accessibilityHint={t(ContactSheetTexts.customer_service.a11yHint)}
           onPress={() => {
             Linking.openURL(customer_service_url);
             close();
           }}
-          iconPosition="right"
-          icon={() => (
-            <ThemeIcon colorType="primary_2" svg={Support}></ThemeIcon>
-          )}
         />
 
-        {enable_intercom ? (
-          <View {...screenReaderHidden}>
-            <View style={styles.descriptionSection}>
-              <ThemeText type="body__secondary" color="secondary">
-                {t(ContactSheetTexts.intercom.title)}
-              </ThemeText>
-              <ThemeText>{t(ContactSheetTexts.intercom.body)}</ThemeText>
-            </View>
+        {customer_feedback_url ? (
+          <ContactItem
+            title={t(ContactSheetTexts.customer_feedback.title)}
+            body={t(ContactSheetTexts.customer_feedback.body)}
+            buttonText={t(ContactSheetTexts.customer_feedback.button)}
+            accessibilityHint={t(ContactSheetTexts.customer_feedback.a11yHint)}
+            onPress={() => {
+              Linking.openURL(customer_feedback_url);
+              close();
+            }}
+          />
+        ) : undefined}
 
-            <Button
-              color="primary_2"
-              text={t(ContactSheetTexts.intercom.button)}
-              accessibilityHint={t(ContactSheetTexts.intercom.a11yHint)}
-              onPress={() => {
-                unreadCount
-                  ? Intercom.displayMessenger()
-                  : Intercom.displayConversationsList();
-                close();
-              }}
-              iconPosition="right"
-              icon={() =>
-                unreadCount ? (
-                  <ThemeIcon colorType="primary_2" svg={ChatUnread} />
-                ) : (
-                  <></>
-                )
-              }
-            />
-          </View>
+        {enable_intercom ? (
```

Comment:
Kunne man endret logikken her litt slik at man ikke risikerer å få opp `ContactItem` for både Intercom og feedback-urlen? De har jo også samme tekster, så det gir ikke mening å vise begge.

Reply:
Endret nå.